database/migrations/2021_02_27_175400_view_minor_pii_data.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+use App\MedusaConfig;
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class ViewMinorPiiData extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+	$config = [
+	    'default' => 18,
+	    'data' => [
+	        [
+		    'field' => 'state_province',
+		    'value' => 'AL',
+		    'age' => 19
+		],
+	        [
+		    'field' => 'state_province',
+		    'value' => 'NE',
+		    'age' => 19
+		],
+	        [
+		    'field' => 'state_province',
+		    'value' => 'MS',
+		    'age' => 21
+		]
+	    ]
+	];
+
+        MedusaConfig::set('minor_pii_config', $config);
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+	MedusaConfig::remove('minor_pii_config');
+    }
```

Comment:
@cinnion Add audit trail to the up and down methods

Reply:
Pushed.